tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -1165,6 +1480,24 @@ static struct bt_conn_auth_info_cb auth_info_cb = {
 	.pairing_complete = auth_pairing_complete,
 };
 
+#if defined(CONFIG_BT_CLASSIC)
+static void auth_pincode_entry(struct bt_conn *conn, bool highsec)
+{
+	const char *pin = "0000";
+	const char *pin16 = "0000000000000000";
```

Comment:
this suggests that there might be something missing on BTP side (for now I'm fine with fixed pins since I suppose this can be set via IXIT on PTS side)

should there be event + cmd for this? similar to passkey entry?